dd-sdk-android/src/main/java/com/datadog/opentracing/DDTracer.java

```diff
@@ -122,9 +122,9 @@ protected DDTracer(final Config config, final Writer writer, final Random random
                 config.getServiceName(),
                 writer,
                 Sampler.Builder.forConfig(config),
-                HttpCodec.createInjector(Config.get()),
```

Comment:
This looked like a bug in the original lib, would live your thoughts on it @xgouchet.